web/annotation_editor_layer_builder.css

```
@@ -189,11 +183,108 @@
   transition-delay: var(--loading-icon-delay);
 }
 
-.annotationEditorLayer .stampEditor.selectedEditor {
-  resize: horizontal;
+.annotationEditorLayer .stampEditor canvas {
+  width: 100%;
+  height: 100%;
 }
 
-.annotationEditorLayer .stampEditor canvas {
+.annotationEditorLayer .resizers {
   width: 100%;
   height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
```

Nit: Could we achieve the same thing the following way, or does that not work for reasons I'm overlooking?
```suggestion
  inset: 0;
  position: absolute;
```